visualisation/map.js

```diff
@@ -220,18 +220,30 @@ map.on('load', function () {
 });
 
 
-var toggleableLayerIds = ['local-authority-boundaries', 'known-groups', 'deprivation', 'pop_density', 'pop_elderly', 'language', 'groupsCount', 'groupsCount_pop', 'groupsCount_elderly'];
+var toggleableLayers = [
+    { id: 'local-authority-boundaries', name: 'Local authority boundaries' },
+    { id: 'known-groups', name: 'Known groups' },
+    { id: 'deprivation', name: 'Index of multiple deprivation' },
+    { id: 'pop_density', name: 'Population density' },
+    { id: 'pop_elderly', name: 'Elderly population' },
+    { id: 'language', name: 'Language' },
+    { id: 'groupsCount', name: 'Number of groups' },
+    { id: 'groupsCount_pop', name: 'Number of groups by population' },
+    { id: 'groupsCount_elderly', name: 'Number of groups by elderly population' }
+];
```

Comment:
If I am not mistaken, it looks like that the same information are also reported in the `layers` array (see [here](https://github.com/DynamicGenetics/Covid-Communities-Map/pull/2/commits/039b6daf54e708a4dddbafcf929d3e354c6dc4f4))

So, I guess we could use that to get and feed `id` and `name` properties in links.
WDYT?

Reply:
In my idea, having one single array would simplify the management of layers. 
On one hand, I reckon there was a `toggleableLayersIds` list which presumably controlled the list of layers we want to be toggle-able. 

However, imho: (a) in case, we could add an extra _ad-hoc_ property in the objects in `layers` to control this (also paired with `shownByDefault` to determine the status of the checkbox menu?); (b) if we don't want some layers to appear on map, there is no need to hard-code them anyway.

my2¢ ;)

Reply:
Thanks @leriomaggio ! I *think* that the piece you are mentioning was overwritten in a later commit, so that it does what you are describing? I can't work out how to link it to this comment! I.e. it feeds through the id and name properties of all the layers, to avoid hard coding them in a `toggableLayers` list for the menu.  ([Line 256 onwards here ](https://github.com/DynamicGenetics/Covid-Communities-Map/pull/2/files?file-filters%5B%5D=.css&file-filters%5B%5D=.js#) ) Very possible I have misunderstood though

Reply:
oooh that's the **outdated** banner GitHub was waving as a _warning_ to me to say .. "hey, you need another coffee, right now!" 🤦 

I _think_ I simply reviewed the commits in reverse order 🙄 (from latest up to earliest!!!)
Sorry! 

Glad that you agree with me, though 😂